wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/Components/ConversationSenderMessageDetailsCell.swift

```diff
@@ -168,88 +160,102 @@ class ConversationSenderMessageDetailsCell: UIView, ConversationMessageCell {
 
+        default:
+            break
         }
 
         switch object.teamRoleIndicator {
 
         case .guest:
             accessibilityIdentifier = "img.guest"
             attributedString.append(
-                stringForAttachment(
-                    named: .guest,
-                    imageSize: 12
+                attachment(
+                    from: .guest,
+                    size: 14
                 )
             )
         case .externalPartner:
             accessibilityIdentifier = "img.externalPartner"
             attributedString.append(
-                stringForAttachment(
-                    named: .externalPartner,
-                    imageSize: 12
-
+                attachment(
+                    from: .externalPartner,
+                    size: 16
                 )
             )
         case .federated:
             accessibilityIdentifier = "img.federatedUser"
             attributedString.append(
-                stringForAttachment(
-                    named: .federated,
-                    imageSize: 12
+                attachment(
+                    from: .federated,
+                    size: 14
                 )
             )
         case .service:
             accessibilityIdentifier = "img.serviceUser"
             attributedString.append(
-                stringForAttachment(
-                    named: .bot,
-                    imageSize: 12
+                attachment(
+                    from: .bot,
+                    size: 14
                 )
             )
 
         default:
             accessibilityIdentifier = "img.member"
         }
 
+        let paragraphStyle = NSMutableParagraphStyle()
+        paragraphStyle.maximumLineHeight = UIFont.mediumSemiboldFont.lineHeight
+
+        attributedString.addAttribute(.paragraphStyle, value: paragraphStyle, range: attributedString.wholeRange)
+
         authorLabel.attributedText = attributedString
     }
 
-    private func stringForAttachment(named imageName: StyleKitIcon, imageSize: CGFloat) -> NSAttributedString {
+    private func attachment(from icon: StyleKitIcon, size: CGFloat) -> NSAttributedString {
         let textColor: UIColor = SemanticColors.Icon.foregroundDefault
         let attachment = NSTextAttachment()
-        let image = imageName.makeImage(
-            size: StyleKitIcon.Size(floatLiteral: imageSize),
+
+        let icon = icon.makeImage(
+            size: StyleKitIcon.Size(floatLiteral: size),
             color: textColor
         ).with(insets: UIEdgeInsets(top: 0, left: 8, bottom: 0, right: 0), backgroundColor: .clear)
 
-        attachment.image = image
+        let iconSize = icon?.size
+
+        let iconBounds = CGRect(x: CGFloat(0),
+                                y: (UIFont.mediumSemiboldFont.capHeight - iconSize!.height) / 2.0,
+                                width: iconSize!.width,
+                                height: iconSize!.height)
+        attachment.bounds = iconBounds
```

Comment:
It would be better to do a guard let for the image so you don't crash a few lines down when force unwrapping.